Directory.Build.props

```diff
@@ -42,7 +42,7 @@
     <Product>aspnet-contrib</Product>
     <Company>$(Authors)</Company>
     <_ProjectCopyright>© AspNet.Security.OAuth.Providers contributors. All rights reserved.</_ProjectCopyright>
-    <PackageIcon>package-icon.png</PackageIcon>
+    <PackageIconFullPath>$(MSBuildThisFileDirectory)package-icon.png</PackageIconFullPath>
```

Comment:
Dunno if it's related to this particular change, but our icon now appears twice in each package:

![image](https://user-images.githubusercontent.com/6998306/70853653-2fe80180-1eb1-11ea-9bf1-f166c938ec8f.png)

Reply:
Yeah, I noticed that too.

With what was there before it complained that it couldn’t add the .NET logo as we had already specified ours, and if I overrode this and left before then it complained about there still being two logos.

Maybe I’ll try reverting back and blanking out this new one and seeing if it goes away.

This might be another one of those annoying hard-to-override arcade things again like we had with the build number.

Reply:
Fix it - Arcade adds the relevant item to include the icon itself by convention somehow, so I just needed to remove the line where I was adding ours manually.

Reply:
Thanks!

Reply:
BTW, is that property 100% required? Looks like EF was updated to not set it explicitly (as you said, it's likely some sort of convention used by Arcade): https://github.com/dotnet/efcore/pull/18805/files

Reply:
If I remember correctly, this is so it doesn't use the default Microsoft icon.

Unless if there's a specific file name our icon could be instead and then it'd "just work".

Reply:
Just tried renaming it to `packageIcon.png` in the root, and we get the .NET icon instead.

Reply:
Perfect, thanks! 😄